.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -248,13 +248,13 @@
 
     <Target Name="ProcessMauiAssets">
         <ItemGroup Condition="'$(_ResizetizerIsAndroidApp)' == 'True'">
-            <AndroidAsset Include="@(MauiAsset)" Link="%(FileName)%(Extension)" />
+            <AndroidAsset Include="@(MauiAsset)" Link="%(RecursiveDir)\%(FileName)%(Extension)" />
```

Comment:
If I put this in my `.csproj`:
```xml
<MauiAsset Include="My\Sub\Folder\Foo.mp3" />
```

It would still evaluate to:
```
<AndroidAsset Include="@(MauiAsset)" Link="Foo.mp3" />
```

I think the only way to fix this up properly is to write an MSBuild task that:

* If it is an absolute path, remove the first part of the path to make it relative. This might get complicated with project references.
* For relative paths, we can probably use them as-is.

Does this fix help another team like Blazor? We could merge this as-is, because it improves things for `**\*` at least.

Reply:
I don't think we need to rush this in for them just yet, we can take a bit longer to fix it 'properly'.  But yes, the idea is Blazor needs to keep the folder hierarchy in tact.